reader_core/src/draw.rs

```diff
@@ -143,6 +216,45 @@ pub fn draw_pkx(pkx: &impl Pkx) {
     print_stat!(iv_spe, ev_spe, Spe, &nature_stat, "Spe ");
 }
 
+pub fn draw_controls_help() {
+    pnp::println!("[Start] + [Select]:");
+    pnp::println!(" - Pause Game");
+    pnp::println!("");
+    pnp::println!("[Start]/[A]:");
+    pnp::println!(" - Unpause Game");
+    pnp::println!("");
+    pnp::println!("[Select]:");
+    pnp::println!(" - Frame Advance");
+}
+
+pub fn draw_specific_help(draw_func: fn() -> ()) {
+    draw_func();
+}
+
+pub fn draw_misc_help() {
+    pnp::println!("PokeReader");
+    draw_version();
+    pnp::println!("");
+    pnp::println!("Current Title:");
+    print_title();
+    pnp::println!("");
+    pnp::println!("PokemonRNG Resources:");
+    pnp::println!(color=0xd7005f, "  pokemonRNG.com");
```

Comment:
Let's capitalize the start of the site:
```suggestion
    pnp::println!(color=0xd7005f, "  PokemonRNG.com");
```